pkg/tcpip/link/tun/device.go

```diff
@@ -264,13 +275,6 @@ func (d *Device) encodePkt(info *channel.PacketInfo) (buffer.View, bool) {
 		vv.AppendView(buffer.View(hdr))
 	}
 
```

Comment:
This check is a bug, as with TUN interface, info.Pkt.LinkHeader().View().IsEmpty() is true by design as well as len(info.Route.RemoteLinkAddress).